src/clib/Make.config.objects

```diff
@@ -39,4 +35,8 @@ OBJS_CONFIG_LIB = \
         set_default_chemistry_parameters.lo \
         solve_chemistry.lo \
         solve_rate_cool_g.lo \
-        update_UVbackground_rates.lo 
+        update_UVbackground_rates.lo \
+        cie_thin_cooling_rate_g_c.lo \
+        rate_functions.lo \
+        initialise_rates.lo \
+        writeRates.lo \
```

Comment:
The `writeRates` function has been removed, right? If so, you can remove that line.